### ISSUE TITLE
Two `h1` tags on question pages.

### ISSUE THREAD

**Issue by hamishtaplin** [ONSdigital/eq-survey-runner#2033]
### Expected behaviour

For accessibility reasons, there should only be a single `h1` on the page. This should be the most relevant title possible (eg. the question title).

### Actual behaviour

On question pages, there are two `h1`:
1. the title of the questionnaire (in the banner)
1. the question title

### Steps to reproduce the behaviour

1. Look at any questionnaire
1. inspect the header, the title is in a `h1`
1. inspect the question title, it is a `h1`

**Comment by hamishtaplin** [comment]
This defect was introduced in [this PR](https://github.com/ONSdigital/eq-survey-runner/commit/2556e5a6b6deeb36f2b1c8448c2c7b926c090500#diff-768d955cccee81fc7eaae9594f7f8bb0). 

The intro page used to have a `topbar-introduction` partial which contained the `h1` for that page only. Subsequent pages used a different topbar with no `h1`.

**Comment by hamishtaplin** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Two H1s on every page](https://trello.com/c/am5DskPa/1016-two-h1s-on-every-page)
